base/linalg/dense.jl

```diff
@@ -112,6 +112,43 @@ true
 isposdef(A::AbstractMatrix) = ishermitian(A) && isposdef(cholfact(Hermitian(A)))
 isposdef(x::Number) = imag(x)==0 && real(x) > 0
 
+"""
+    stride1(A) -> Int
+
+Returns the distance between successive array elements
```

Comment:
Perhaps take some inspiration from the `stride` docs? E.g. something like
```
Return the distance in memory (in number of elements) between adjacent
elements in dimension 1.
```

Reply:
Yes, I used the stride docs as a reference.

Reply:
I meant that perhaps it should be noted in the docs that it is along dimension 1.

Reply:
Thank you very much for your kind instruction.
I've modified explanation of return value. Does it make sense?